README.md

```diff
@@ -94,6 +94,19 @@ Fluid Dialogue is used through [Unity's Package Manager](https://docs.unity3d.co
 
 The following APIs are available for customizing your dialogue experience.
 
+### Initializing a Dialogue
+
+You can initialize a dialogue using the .Play method, as show in the following script : 
+https://github.com/ashblue/fluid-dialogue/blob/develop/Assets/Examples/BasicConversation/Scripts/ExampleDialoguePlayback.cs#L56
+
+```
+   private void Awake () {
```

Comment:
Adding in all the logic to run this to minimize the amount of digging necessary to run it.

```suggestion
using System.Collections;
using System.Linq;
using CleverCrow.Fluid.Databases;
using CleverCrow.Fluid.Dialogues.Graphs;
using UnityEngine;

public class ExampleDialoguePlayback : MonoBehaviour {
  private DialogueController _ctrl;
  public DialogueGraph dialogue;
  public GameObjectOverride[] gameObjectOverrides;

  private void Awake () {
```